### PR TITLE
D2k fix for cliff passability

### LINKED ISSUE #9166: Carryalls disappear under FoW in d2k
This is on bleed. The only problem I have with this is that the carryall makes off with my harvester and unless I am constantly defending my spice patches, I can't see if someone set up a rocket tank to snipe the carryalls.
## <bountysource-plugin>

Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/26174870-carryalls-disappear-under-fow-in-d2k?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by reaperrr**
Fixes passability on some d2k cliffs.

### CODE REVIEW

File: mods/d2k/rules/defaults.yaml [hunk 1]
```diff
@@ -237,7 +239,7 @@
 	Inherits@1: ^ExistsInWorld
 	AppearsOnRadar:
 		UseLocation: yes
-	HiddenUnderFog:
+	HiddenUnderShroud:
```
Comment (pchote) [inline mods/d2k/rules/defaults.yaml]: This makes them visible for *everyone* under the fog, but we want them to be visible to only the owner and their allies.

Comment (reaperrr) [inline mods/d2k/rules/defaults.yaml]: Oh, I see.

**Comment by pchote** [comment]
After refreshing my memory on the history behind #9166, I think we shouldn't fix it.

**Comment by reaperrr** [comment]
Updated.

> After refreshing my memory on the history behind #9166, I think we shouldn't fix it (its a feature, not a bug).

#9106 wants to "fix" this as well though, meaning we've now had at least two people who considered it a bug (or at least inconvenience), so we should at least keep an eye on this.

**Comment by GraionDilach** [comment]
Although that PR also has this included, but I guess this can go without that so :+1:

**Comment by reaperrr** [comment]
It may take a while until that is mergable, and this here might still make it into the playtest/release.

**Comment by Mailaender** [comment]
Thanks.

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/a1bc3e252cb73bc5347be9a5997ff224aae4990d...5064e96a77cbf5d3d3ef0acdcdd10e44d2ccb4b2)

**Pull request merged**